k8s/README.md

```diff
@@ -0,0 +1,7 @@
+This directory includes the Kubernetes configuration for deploying this
+website, which uses Tekton to:
+
+- Clone the website from the tektoncd/website repository
+- Synchronizes contents from specified Tekton repositories
+- Build the website with Hugo
+- Deploys thew website via Firebase
```

Comment:
since we're using netlify now, i'm thinking maybe we don't need these?

Reply:
I'll remove this part.